index.html

```diff
@@ -0,0 +1,771 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Balsamiq+Sans:ital,wght@0,400;0,700;1,400;1,700&family=Fondamento:ital@0;1&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/9a2f12287f.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Smashing Magazine</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <ul class="flex justify-content-space-around align-items">
+          <li>
+            <img
+              src="https://www.smashingmagazine.com/images/logo--full.svg"
+              alt="Smashing Magazine Icon"
+            />
+          </li>
+          <li><button>Articles</button></li>
+          <li><button>Guides</button></li>
+          <li><button>Books</button></li>
+          <li><button>Events</button></li>
+          <li><button>Membership</button></li>
+          <li><button>Jobs</button></li>
+          <li>
+            <button class="header-more-button">
+              More <i class="fas fa-frog"></i>
+            </button>
+          </li>
+          <li><input type="text" placeholder="Search articles..." /></li>
+        </ul>
+      </nav>
+    </header>
+    <div class="headline flex">
+        <div class="headline-left"></div>
+        <div class="headline-right"></div>
+        <h2 class="top-four-articles-headline">
+            <span>Don't Miss These Latest</span><br /><span>Articles</span>
+      </h2>
+    </div>
+    <section class="top-four-articles">
+      <div class="grid top-four-articles-grid">
+        <article class="top-left flex">
+          <a href="#" class="flex">
+            <div class="small-transition-box rotate-on-hover"></div>
+            <p class="top-four-article-author">
```

Comment:
- [ ] **[OPTIONAL]** For good HTML semantics, the `<main>` tag is normally used for the dominant content of the `<body>` of a document. In this case, if you want to implement it, you could use it instead of your `<section class="top-four-articles">`

Reply:
added main tag